src/Symfony/Bundle/TwigBundle/Controller/ExceptionController.php

```diff
@@ -64,7 +64,7 @@ public function showAction(Request $request, FlattenException $exception, DebugL
         $code = $exception->getStatusCode();
 
         return new Response($this->twig->render(
-            (string) $this->findTemplate($request, $request->getRequestFormat(), $code, $showException),
+            $this->findTemplate($request, $request->getRequestFormat(), $code, $showException),
```

Comment:
`findTemplate()` returns a string, the returned var always comes from a `sprintf()`.